src/prefix-for-network.ts

```diff
@@ -19,7 +19,7 @@ export = function getPrefixForNetwork(network: string): string {
       prefix = 'kovan.'
       break
     default:
-      prefix = ''
+      prefix = 'other.'
```

Comment:
Might it make sense to return `null` instead of a valid prefix that isn't meant to be used?

Reply:
It would change the type signature, but that seems OK to me. This function isn't exposed in the package's API, so it wouldn't be a breaking change. Returning `null` here would be less confusing I think.